sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py

```diff
@@ -79,7 +72,7 @@ def score_data(
     (Path(score_output) / "score").write_text("scored with {}".format(model))
 
 
-@command_component(display_name="Evaluate", environment=conda_env)
+@command_component(display_name="Evaluate", environment="./conda.yaml")
```

Comment:
rename to env.yaml like the previous pr?

Reply:
renamed